extensions/adapters/vector/src/main/java/mil/nga/giat/geowave/adapter/vector/util/FeatureDataUtils.java

```diff
@@ -76,4 +82,56 @@ else if (crs != null) {
 		}
 		return defaultCRSEntry;
 	}
+
+	public static String getAxis(
+			final CoordinateReferenceSystem crs ) {
+		final CoordinateSystem cs = crs == null ? null : crs.getCoordinateSystem();
+		if (cs != null && cs.getDimension() > 0) return cs.getAxis(
+				0).getDirection().name().toString();
+		return "";
```

Comment:
Thinking it over a bit and having a quick discussion with Rich, decided to stick with our default, EPSG:4326 with Axis EAST.